sentry_sdk/crons/decorator.py

```diff
@@ -5,7 +5,7 @@
 from sentry_sdk.utils import now
 
 if TYPE_CHECKING:
-    from typing import Optional, Type
+    from typing import Any, Dict, Optional, Type
```

Comment:
```suggestion
    from typing import Any, Optional, Type
```

We can use built-in `dict` (`typing.Dict` is a deprecated alias to `dict`).